docs/usage/config-presets.md

```diff
@@ -95,21 +96,23 @@ You can set a Git tag (like a SemVer) to use a specific release of your shared c
 
 ## Example configs
 
-An example of a small rule is `:preserveSemverRanges`, which has the description "Preserve (but continue to upgrade) any existing semver ranges".
+An example of a small rule is `:preserveSemverRanges`, which has the description "Preserve (but continue to upgrade) any existing SemVer ranges.".
 It simply sets the configuration option `rangeStrategy` to `replace`.
 
 An example of a full config is `config:base`, which is Renovate's default configuration.
-It mostly uses Renovate config defaults but adds a few smart customisations such as grouping monorepo packages together.
+It mostly uses Renovate config defaults but adds a few smart customizations such as grouping monorepo packages together.
 
 <!-- prettier-ignore -->
 !!! note
-    The `:xyz` naming convention (with `:` prefix) is a special shorthand for the `default:` presets.
-    e.g. `:xyz` is equivalent to `default:xyz`.
+    The `:xyz` naming convention (with `:` prefix) is shorthand for the `default:` presets.
+    For example: `:xyz` is the same as `default:xyz`.
 
 ## How to Use Preset Configs
 
-By default, the Renovate App's onboarding process will suggest `["config:base]"`.
-If you are self hosting you must add `"onboardingConfig": { "extends": ["config:base"] }` to your bot's config.
+By default, Renovate App's onboarding PR suggests the `["config:base]"` preset.
+If you're self hosting you must add `"onboardingConfig": { "extends": ["config:base"] }` to your bot's config.
```

Comment:
> If you're self hosting you must add `"onboardingConfig": { "extends": ["config:base"] }` to your bot's config.

The current wording makes it sound like people _must_ use our `config:base` preset for some reason. 😄 

I think we mean to say:

```suggestion
If you're self hosting, and want to use the `config:base` preset, then you must add `"onboardingConfig": { "extends": ["config:base"] }` to your bot's config.
```